src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -485,6 +485,9 @@ public virtual async Task<bool> UpdateTopologyAsync(UpdateTopologyParameters par
                 using (ContextPool.AllocateOperationContext(out JsonOperationContext context))
                 {
                     var command = new GetDatabaseTopologyCommand(parameters.DebugTag, Conventions.SendApplicationIdentifier ? parameters.ApplicationIdentifier : null);
+                    if (DefaultTimeout.HasValue)
+                        command.Timeout = new[] { DefaultTimeout, command.Timeout }.Max();
```

Comment:
Let's avoid LINQ, probably can change that to:
```
                    if (DefaultTimeout.HasValue && DefaultTimeout.Value > commad.Timeout)
                        command.Timeout = DefaultTimeout.Value;
```

Would that work?

Reply:
Done